setup.py

```diff
@@ -1,14 +1,25 @@
 import versioneer
 from setuptools import setup
 
-setup(name='psdm_qs_cli',
-      version=versioneer.get_version(),
-      cmdclass=versioneer.get_cmdclass(),
-      description='Python client for interacting with the PCDS questionnaire.',
-      url='https://github.com/slaclab/psdm_qs_cli.git',
-      author='Murali Shankar',
-      author_email='mshankar@slac.stanford.edu',
-      license='MIT',
-      packages=['psdm_qs_cli'],
-      scripts=['psdm_qs_cli/QSGenerateExcelSpreadSheet.py', 'psdm_qs_cli/QSGenerateJSON.py',],
-      zip_safe=False)
+with open('requirements.txt', 'rt') as f:
+    requirements = f.read().splitlines()
+
+setup(
+    name="psdm_qs_cli",
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
+    description="Python client for interacting with the PCDS questionnaire.",
+    url="https://github.com/slaclab/psdm_qs_cli.git",
+    author="Murali Shankar",
+    author_email="mshankar@slac.stanford.edu",
+    license="MIT",
+    packages=["psdm_qs_cli"],
+    entry_points={
+        "console_scripts": [
+            "QSGenerateExcelSpreadSheet.py=psdm_qs_cli.QSGenerateExcelSpreadSheet:main",
+            "QSGenerateJSON.py=psdm_qs_cli.QSGenerateJSON:main",
+        ],
```

Comment:
If we could track down who is using the scripts we could probably switch them to remove the `.py` at the name? 
This is just aesthetics... feel free to ignore

Reply:
Yeah, I'd consider this back-compat. No clue who is using these.

Reply:
Yeah, we better keep the same name! 😉